### PR TITLE
emacs, emacsclient: refresh

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/common/emacs.md [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+`emacs {{path/to/file}}`
 
-- Open a file:
+- Open a file at a specified line number:
 
-`emacs {{path/to/file}}`
+`emacs +{{line_number}} {{path/to/file}}`
 
-- Save a file:
+- Start Emacs in console mode (without X window):
```
Comment (bl-ue) [inline pages/common/emacs.md:15]:
```suggestion
- Start Emacs in console mode (without an X window):
```

**Review by bl-ue — COMMENTED** [review]
Hi! A few comments... :)

File: pages/common/emacs.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/emacs.md:15]: I like `without X window` however we'll see

Comment (marchersimon) [reply pages/common/emacs.md:15]: `X window` isn't really a name, but a noun, so it should be `the X window`/`an X window`.

**Comment by CleanMachine1** [comment]
May I ask, what was https://github.com/tldr-pages/tldr/pull/6096/commits/0242960b31ca322a7b56aadca7168dcbb76e3eb2 for?

**Comment by bl-ue** [comment]
How'd you get that?

**Comment by CleanMachine1** [comment]
It was the first commit for this branch.

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
I've just realised, that was probably a `git commit -m "test" --allow-empty` I use that flag all the time for testing locally.

**Comment by marchersimon** [comment]
> May I ask, what was [0242960](https://github.com/tldr-pages/tldr/commit/0242960b31ca322a7b56aadca7168dcbb76e3eb2) for?

I made a test branch and forget to go back to main, before creating this branch.

> I've just realised, that was probably a `git commit -m "test" --allow-empty` I use that flag all the time for testing locally.

I didn't use any flag. Just created a new empty file and committed it.

**Comment by CleanMachine1** [comment]
Well that flag is pretty useful when you can't be bothered to test by making new files. I don't know how many know about it. I am going to add it to git-commit later

**Review by sbrl — APPROVED** [review]
Awesome work, @marchersimon!

**Review by CleanMachine1 — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**